decidim/decidim
# Add copy to clipboard feature to share links

#### :tophat: What? Why?
We've received user feedback that the copy share link feature is unclear because it shows the "not allowed" cursor for the text field.

This fixes that issue and adds a "copy to clipboard" button to the share link window.

#### Testing
- Go to a proposal page
- Click the share link
- Click the "Share link" button
- See the "not allowed" cursor on top of the input

#### :clipboard: Checklist

### :camera: Screenshots
Now it looks like this with the cursor:
![Share link text field cursor](https://user-images.githubusercontent.com/864340/112216133-d4ff9700-8c29-11eb-9ed8-21b63d60ab55.png)

After this change it looks like this:
![Share link modal after the change](https://user-images.githubusercontent.com/864340/112216195-e34db300-8c29-11eb-9c3e-68784237ceb1.png)

In addition, it will announce the screen reader that the link was copied to clipboard.
